README.md

```diff
@@ -40,7 +40,7 @@ The technical architecture of Apache APISIX:
 
 - Mailing List: Mail to dev-subscribe@apisix.apache.org, follow the reply to subscribe the mailing list.
 - QQ Group - 578997126
-- [Slack Workspace](http://s.apache.org/slack-invite) - join `#apisix` on our Slack to meet the team and ask questions
+- [Slack Workspace](https://join.slack.com/t/the-asf/shared_invite/zt-lcbw4olf-_5s17RLU0Cx8xTvj2bNbjg) - join `#apisix` on our Slack to meet the team and ask questions
```

Comment:
Hi, thanks for this user wall! But we need to update 2 logos:

- DaoCloud: https://static.apiseven.com/logo/daocloud.png
- HelloTalk: https://static.apiseven.com/logo/HelloTalk.png